sktime/forecasting/tests/test_all_forecasters.py

```diff
@@ -57,17 +57,32 @@
 y_train, y_test = temporal_train_test_split(y, train_size=0.75)
 
 
+# helper function
+def _get_n_columns(tag):
```

Comment:
Could we add a docstring or comment here describing why we need this, i.e. to test forecasters which are univariate, multivariate or both

Reply:
I will include it